src/EditorFeatures/CSharpTest/SymbolKey/SymbolKeyErrorTypeTests.cs

```diff
@@ -0,0 +1,408 @@
+                    }
+
+                    public struct S
+                    {
+                    }
+                }
+                """;
+
+            VerifyResolution(source, c => c.GetMember("N.C.M"));
+        }
+
+        [Fact]
+        public void NestedType_MissingInGenericContainer()
+        {
+            var source = """
+                using System.Collections.Generic;
+
+                namespace N
+                {
+                    public class C
+                    {
+                        public void M(List<int>.Enumerator x)
+                        {
+                        }
+                    }
+                }
+                """;
+
+            VerifyResolution(source, c => c.GetMember("N.C.M"));
+        }
+
+        [Fact]
+        public void NestedType_MissingInNonGenericContainer()
+        {
+            var source = """
+                using System.Diagnostics;
+
+                namespace N
+                {
+                    public class C
+                    {
+                        public void M(DebuggableAttribute.DebuggingModes x)
+                        {
+                        }
+                    }
+                }
+                """;
+
+            VerifyResolution(source, c => c.GetMember("N.C.M"));
+        }
+
+        [Fact]
+        public void Method_MissingReturnType()
+        {
+            var source = """
+                namespace N
+                {
+                    public class C
+                    {
+                        public string Create()
+                        {
+                            return new string('c', 1);
+                        }
+                    }
+                }
+                """;
+
+            VerifyResolution(source, c => c.GetMember("N.C.Create"));
+        }
+
+        [Fact]
+        public void Method_MissingParameterType()
+        {
+            var source = """
+                namespace N
+                {
+                    public class C
+                    {
+                        public C Create(string x)
+                        {
+                            return new C();
+                        }
+                    }
+                }
+                """;
+
+            VerifyResolution(source, c => c.GetMember("N.C.Create"));
+        }
+
+        [Fact]
+        public void Constructor_MissingParameterType()
+        {
+            var source = """
+                public class C
+                {
+                    public C(string x)
+                    {
+                    }
+                }
+                """;
+
+            VerifyResolution(source, c => c.GetMember("C..ctor"));
+        }
+
+        [Fact]
+        public void Indexer_MissingParameterType()
+        {
+            var source = """
+                namespace N
+                {
+                    public class C
+                    {
+                        public C this[string x]
```

Comment:
Does this count as a parameterized property, or do I have to create a VB test file ?

Reply:
Yup. This counts